tests/test_trainer.py

```diff
@@ -143,6 +153,7 @@ def get_regression_trainer(a=0, b=0, double_output=False, train_len=64, eval_len
         optimizers = kwargs.pop("optimizers", (None, None))
         output_dir = kwargs.pop("output_dir", "./regression")
         args = TrainingArguments(output_dir, **kwargs)
+        trainer_args = trainer_args or {}
```

Comment:
Can we pop from the kwargs instead (like for optimizers just above?)

Reply:
Yes, of course.

Reply:
Fixed in https://github.com/huggingface/transformers/pull/7935/commits/1c4d60817d922783f8bbac1764674918476dcbd9